dd-trace-api/src/main/java/datadog/trace/api/config/GeneralConfig.java

```diff
@@ -56,6 +56,8 @@ public final class GeneralConfig {
 
   public static final String TELEMETRY_ENABLED = "instrumentation.telemetry.enabled";
   public static final String TELEMETRY_HEARTBEAT_INTERVAL = "telemetry.heartbeat.interval";
+  public static final String TELEMETRY_DEPENDENCY_SERVICE_ENABLED =
```

Comment:
per the spec, the name of the environment variable should be `DD_TELEMETRY_SEND_DEPENDENCIES`.

```suggestion
  public static final String TELEMETRY_SEND_DEPENDENCIES = "telemetry.send.dependencies";
```

Reply:
done